### PR TITLE
feat(*): on disable active flag is also off

### PR DESCRIPTION
**Pull request by DZariusz**
When disabeling both flags `enable` and `avtive` go down.
Verifier can't activate itself when disabled.

LIT-287

### CODE REVIEW

File: contracts/VerifierRegistry.sol [hunk 1]
```diff
@@ -159,6 +159,7 @@ contract VerifierRegistry is IVerifierRegistry, Ownable, RegistrableWithSingleSt
 
     require(verifier.id != address(0x0), "verifier do not exists");
     require(verifier.active != _active, "no changes to active flag");
+    require(verifier.enable, "verifier must be enable to become active");
```
Comment (therevoltingx) [inline contracts/VerifierRegistry.sol]: typo should be: `must be enabled`

File: contracts/VerifierRegistry.sol [hunk 2]
```diff
@@ -176,6 +177,9 @@ contract VerifierRegistry is IVerifierRegistry, Ownable, RegistrableWithSingleSt
     require(verifier.enable != _enable, "no changes to enable flag");
 
     vrStorage.setVerifierEnable(_verifier, _enable);
+    if (!_enable) {
```
Comment (therevoltingx) [inline contracts/VerifierRegistry.sol]: missing empty line.
remember to add empty line between multi-line statements

File: test/ministro-contracts/ministroVerifierRegistry.js [hunk 3]
```diff
@@ -111,6 +111,9 @@ function MinistroContract() {
       const savedVerifier = await app.verifiers(verifier);
       assert(areAddressesEqual(savedVerifier.id, verifier), 'should be saved onchain');
       assert.strictEqual(savedVerifier.enable, enable, 'invalid enable status');
+      if (!enable) {
```
Comment (therevoltingx) [inline test/ministro-contracts/ministroVerifierRegistry.js]: remember to add empty line before multi-line statements

File: test/unit/TestVerifierRegistry.js [hunk 4]
```diff
@@ -125,12 +117,31 @@ contract('VerifierRegistry', (accounts) => {
       );
     });
 
-    it('should be able to update enable state by owner', async () => {
-      await ministroVerifierRegistry.updateEnableStatus(
-        verifier,
-        true,
-        { from: owner },
-      );
+    describe('when verifier enable', () => {
```
Comment (therevoltingx) [inline test/unit/TestVerifierRegistry.js]: is this supposed to be `when verifier is enabled` your tests should read like sentences

File: test/unit/TestVerifierRegistry.js [hunk 5]
```diff
@@ -125,12 +117,31 @@ contract('VerifierRegistry', (accounts) => {
...
+      before(async () => {
+        await ministroVerifierRegistry.updateEnableStatus(
+          verifier,
+          true,
+          { from: owner },
+        );
+      });
+
+      it('should NOT be able to update active status by not a verifier', async () => {
```
Comment (therevoltingx) [inline test/unit/TestVerifierRegistry.js]: fix language here, it's weird

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,7 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 ### Added
-- option to enable and disable verifier (`active` flag): 
+- option to enable and disable verifier (`enable` flag):
```
Comment (therevoltingx) [inline CHANGELOG.md]: should be `enabled`

Comment (DZariusz) [reply CHANGELOG.md]: damn... docs

**Comment by therevoltingx** [comment]
👍

**Pull request merged**